docs/items/weapon-configs.md

```diff
@@ -433,34 +445,42 @@ What values a Weapon Type can have:
     * Direct Damage Scalar: 1.55
     * Grenade Damage Scalar: 1.00
     * Explosive Damage Scalar: 1.00
+  * VFX: VIP
```

Comment:
The VIP and Infected VFX aren't traits so lets put these under the scripted behavior